schemas/ep00_epics_connection_info.fbs

```diff
@@ -0,0 +1,23 @@
+// Represent events about the underlying EPICS connection.
+
+file_identifier "ep00";
+
+enum EventType: ushort {
+  UNKNOWN,
+  NEVER_CONNECTED,
+  CONNECTED,
+  DISCONNECTED,
+  DESTROYED,
+}
+
+table EpicsConnectionInfo {
+  // Nanoseconds since UNIX epoch
+  timestamp: ulong;
+  type: EventType;
+  // The channel name, called `source_name` to stay in sync with `f142`
+  source_name: string;
+  // Identifies the client which has observed the event
+  service_id: string;
```

Comment:
The comment in `f142` schema says about `forwarder_info` that "optional, for development and debug used by EPICS forwarder" and currently it is no longer used because it was meant for testing before the arrival of system tests. This schema does not need it.

`service_id` is something which one could consider adding to other schemas as well. Adding is always possible without breaking binary compatibility. On the other hand, it's maybe not needed urgently. Here, I thought it's convenient to have, and the expected data rate for this schema is so low that we can easily afford the extra bytes.

Reply:
Thanks for clarifying the forwarder_info. We should consider removing it then. The fields in there are also not terribly well documented.

I can't think of any strong use case for the service_id on the receiving side. In all cases where I might want that information, going to the log files might be a sensible options.

Reply:
`forwarder_info` in schema `f142` was only for testing in the very early phase.
I'll mark it deprecated there, in another PR.

Regarding `service_id`: when you say that you would go to the log output if you'd like that information, what would you look for? EPICS connection changes are not written to the application log on the default log level, and the application log should not be considered a source for that information. The Kafka topic already is.

If we think that knowing the `service_id` on such a EPICS connection change event might be useful, then I think it belongs in this schema.

Reply:
I agree this is the place to put it if we need it, but the "think ... might be useful" makes me uneasy -> YAGNI. As you say, fields can easily be added later if we find we have a solid use case.

Reply:
Well, if you find a disconnect the information where it was encountered is always important and must be conveniently accessible. Consider the use case when you try to debug startup issues: then the information when a service connects to an IOC is important. Both use cases would be covered by the service_id. These messages will be rare, the service_id is a short string, so there is no concern that we fill Kafka with too much data. If this would happen with these messages, we would have a far more serious problem :-;

Reply:
@matthew-d-jones Well, personally I think it is useful and I vote for having it. Use case: Debug a misconfigured NICOS.